packages/synapse-react-client/src/components/dataaccess/SubmissionPage/SubmissionPage.tsx

```diff
@@ -117,8 +117,11 @@ export default function SubmissionPage(props: SubmissionPageProps) {
 
   const { data: currentUserProfile } = useGetCurrentUserProfile()
 
-  const { data: submission, isLoading: isLoadingSubmission } =
-    useGetDataAccessSubmission(submissionId)
+  const {
+    data: submission,
+    isLoading: isLoadingSubmission,
+    error: submissionLoadingError,
+  } = useGetDataAccessSubmission(submissionId)
```

Comment:
An alternative would be to pass `throwOnError` as an option to the useQuery and let a parent error boundary handle it, but I think handling the error explicitly here is fine too

Reply:
I like it!  Updated